src/main/java/uk/ac/cam/cl/dtg/isaac/api/Constants.java

```diff
@@ -42,6 +42,8 @@
     public static final String TOPIC_SUMMARY_PAGE_TYPE = "isaacTopicSummaryPage";
     public static final String EVENT_TYPE = "isaacEventPage";
 
+    public static final String SEARCHABLE_TAG = "searchable";
```

Comment:
Could this be `search_result`? I still regret my choice of `nofilter`, but it's much too hard to change it now. I think `search_result` slightly better explains things.

Reply:
Sure